### DISCUSSION TITLE
SMTP Gmail error

### DISCUSSION THREAD

**Discussion by maci2100** [dani-garcia/vaultwarden#1359] [Q&A]
Hi, if i try to configure the smtp for mm Gmail account the following error shows up: SMTP Client error: No compatible authentication mechanism was found

My Config:
SMTP_HOST: smtp.gmail.com
SMTP_PORT: 587
SMTP_FROM: xxx@gmail.com
SMTP_USERNAME: xxx@gmail.com 
SMTP_PASSWORD: xxx
SMTP_SSL: false

**Comment by BlackDex** [comment]
Please see: https://github.com/dani-garcia/bitwarden_rs/discussions/1360#discussioncomment-342031
